### ISSUE TITLE
Expect sealer to support the request for help   

### ISSUE THREAD

**Issue by cubxxw** [containers/buildah#4579]
### Expect sealer to support the request for help

 [sealer](https://github.com/sealerio/sealer) is a tool for building containers using buildah, I hope it succeeds in building Kubernetes, it currently supports root and sudo modes but not rootless, I hope it supports rootless. I have no idea for this, I hope to get your help ~!

**Comment by flouthoc** [comment]
Hi @cubxxw , Is this issue requesting the sealer supports builds in rootless environment ? If yes could you edit the title since it is not very much clear about what are you trying to report.

**Comment by flouthoc** [comment]
Hi @cubxxw , Could you open same issue at sealer since i think implementation needs to happen at sealer end and buildah maintainers can help sealer maintainers to implement it by resolving any queries they have.

**Comment by cubxxw** [comment]
Of course, thank you very much for your help. I am working on the architecture of sealer and I will try to implement it. I learned that rooless mode mounts must be `buildah unshare`
